docs/admin/projects.rst

```diff
@@ -764,30 +763,29 @@ By default the component is visible to anybody who has access to the project,
 even if the person can not perform any changes in the component. This makes it
 easier to keep translation consistency within the project.
 
-Enable this in case you want to grant access to this component explicitly -
-the project level permissions will not apply and you will have to specify
-component or component list level permission in order to grant access.
+Turning this on at component-, or component-list level takes over
+access permission to a component, regardless of project-level permissions.
 
-Default value can be changed by :setting:`DEFAULT_RESTRICTED_COMPONENT`.
+The default value can be changed in :setting:`DEFAULT_RESTRICTED_COMPONENT`.
 
 .. hint::
 
-   This applies to project managers as well - please make sure you will not
+   This applies to project managers as well — please make sure you will not
    loose access to the component after toggling the status.
 
 .. _component-links:
 
 Share in projects
 +++++++++++++++++
 
-You can choose additional projects where the component will be visible. This
-can be useful for shared libraries which you use in several projects.
+You can choose additional projects where the component will be visible.
+Useful for shared libraries which you use in several projects.
 
 .. note::
 
-   Sharing component doesn't change its access control. It makes it only
-   visible when browsing other projects. User still need to have access to the
-   actual component in order to be able to browse or translate it.
+   Sharing a component doesn't change its access control. It only makes it
+   visible when browsing other projects. Users still need access to the
```

Comment:
```suggestion
   visible when browsing other projects. Users still need an access to the
```

Reply:
To point out it is a privilege based system.

"to be granted access to"
or
"access permission"?

Reply:
I like `need to be granted` more...

Reply:
by the word, even for `need to be granted`, shouldn't there be an article before `access`?

Reply:
please close the discussion once you come to an end with it.